cumulus_library/study_parser.py

```diff
@@ -212,8 +226,12 @@ def _execute_drop_queries(
         :param task: a TaskID for a given progress bar
         """
         for view_table in view_table_list:
+            if explicit_prefix:
+                prefix = f"{view_table[0]}"
+            else:
+                prefix = f"{view_table[0]}__"
             drop_view_table = get_drop_view_table(
-                name=view_table[0], view_or_table=view_table[1]
+                name=f"{view_table[0]}", view_or_table=view_table[1]
```

Comment:
Did you mean to pass in `prefix` here? I don't see it being used.

(And if so, and tests didn't catch this - maybe add a test that would)

Reply:
yeah, good catch and call